app/Http/Controllers/TagAdminController.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
Controller nên đặt theo namespace sẽ tốt hơn là thêm hậu tố Admin vào sau 

Ví dụ: TagAdminController thì nên để là Admin/TagController 

Lúc đó trong folder Http/Controllers sẽ có folder Admin nhóm toàn bộ các Controller của module Admin vào 1 folder

Reply:
vâng.để pull sau e sửa chứ giờ sửa ở pull này sợ lắm file thay đổi quá ạ